src/graphql/data/activity.graphql

```diff
@@ -161,10 +161,10 @@ fragment AssetActivityParts on AssetActivity {
 }
 
 # TODO(UniswapX): return to a pagesize of 50 pre-launch
-query Activity($account: String!) {
-  portfolios(ownerAddresses: [$account]) {
+query Activity($account: String!, $chains: [Chain!]!) {
```

Comment:
non-blocking not: but can we make this optional?

Reply:
If BE defaulted to returning all chains I think that would be best, but since it doesn't, I think it's best to require it so that callers don't accidentally request mainnet-only data without realizing